packages/eas-cli/src/graphql/queries/SubmissionQuery.ts

```diff
@@ -0,0 +1,88 @@
+import { print } from 'graphql';
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import {
+  AppPlatform,
+  GetAllSubmissionsForAppQuery,
+  GetAllSubmissionsForAppQueryVariables,
+  SubmissionFragment,
+  SubmissionStatus,
+  SubmissionsByIdQuery,
+  SubmissionsByIdQueryVariables,
+} from '../generated';
+import { SubmissionFragmentNode } from '../types/Submission';
+
+type Filters = {
+  platform?: AppPlatform;
+  status?: SubmissionStatus;
+  offset?: number;
+  limit?: number;
+};
+
+export const SubmissionQuery = {
+  async byIdAsync(
+    submissionId: string,
+    { useCache = true }: { useCache?: boolean } = {}
+  ): Promise<SubmissionFragment> {
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .query<SubmissionsByIdQuery, SubmissionsByIdQueryVariables>(
+          gql`
+            query SubmissionsByIdQuery($submissionId: ID!) {
+              submissions {
+                byId(submissionId: $submissionId) {
+                  id
+                  ...SubmissionFragment
+                }
+              }
+            }
+            ${print(SubmissionFragmentNode)}
+          `,
+          { submissionId },
+          { requestPolicy: useCache ? 'cache-first' : `network-only` }
```

Comment:
```suggestion
          { requestPolicy: useCache ? 'cache-first' : 'network-only' }
```
I know it doesn't matter though.